shared/provision/error/container.js

```diff
@@ -13,9 +14,8 @@ const mapStateToProps = state => ({
 
 const mapDispatchToProps = (dispatch, ownProps: OwnProps) => ({
   onAccountReset: () => openURL('https://keybase.io/#account-reset'),
-  onBack: () => {
-    !flags.useNewRouter && dispatch(ownProps.navigateUp())
-  },
+  onBack: () =>
+    flags.useNewRouter ? dispatch(RouteTreeGen.createNavigateUp()) : dispatch(ownProps.navigateUp()),
```

Comment:
kinda moot but these can all turn into the global one and never the ownProps.navigateUp